subsys/partition_manager/pm.yml.tfm

```diff
@@ -29,8 +30,9 @@ tfm_secondary:
 app_secondary:
   placement: {after: [tfm_secondary]}
   share_size: app_primary
+#endif
 
 tfm_extra:
-  placement: {after: [app_secondary]}
+  placement: {after: [app_secondary, app]}
```

Comment:
Is this documented in the commit message? Why is this here?

Reply:
Because sometimes app_secondary is not part of the build (if BL2 is disabled)

Reply:
Does this need a comment @hakonfam?

Reply:
No, it makes sense to me now :)